qa/L0_jax_unittest/test.sh

```diff
@@ -4,6 +4,8 @@
 
 set -xe
 
+export NVTE_FUSED_ATTN=1
+
 : ${TE_PATH:=/opt/transformerengine}
 pytest -Wignore -v $TE_PATH/tests/jax -k 'not distributed'
```

Comment:
```suggestion
: ${TE_PATH:=/opt/transformerengine}
NVTE_FUSED_ATTN=1 pytest -Wignore -v $TE_PATH/tests/jax -k 'not distributed'
```

Reply:
Fix in fcf6c564b0d1fd13b834f7f5861df89d199c1976. I moved the env to the specific test files.